backend/requirements.txt

```diff
@@ -1,64 +1,20 @@
-anyio==3.3.4
-asgiref==3.4.1
-astroid==2.8.3
-async-generator==1.10
-attrs==21.2.0
-backports.entry-points-selectable==1.1.0
-black==21.9b0
-bson==0.5.10
-certifi==2021.10.8
-cfgv==3.3.1
-charset-normalizer==2.0.7
-click==8.0.3
-colorama==0.4.4
-coverage==6.0.2
-distlib==0.3.3
-fastapi==0.70.0
-fastapi-pagination==0.9.1
-filelock==3.3.1
-h11==0.12.0
-httpcore==0.13.7
-httptools==0.2.0
-httpx==0.23.0
-identify==2.3.1
-idna==3.3
+anyio==3.6.2
+black==22.3.0
+click==8.0.4
+colorama==0.4.6
+fastapi==0.87.0
+idna==3.4
 iniconfig==1.1.1
-isort==5.9.3
-lazy-object-proxy==1.6.0
 mccabe==0.6.1
 mypy-extensions==0.4.3
 nodeenv==1.6.0
-outcome==1.1.0
-packaging==21.3
-pathspec==0.9.0
-platformdirs==2.4.0
-pluggy==1.0.0
-pre-commit==2.15.0
-py==1.10.0
-pydantic==1.8.2
-pylint==2.11.1
-pyparsing==3.0.1
-pytest==6.2.5
-pytest-asyncio==0.16.0
-pytest-cov==3.0.0
-pytest-mock==3.6.1
-python-dateutil==2.8.2
-python-dotenv==0.19.1
-PyYAML==6.0
+pathspec==0.10.2
+platformdirs==2.5.4
+pydantic==1.10.2
 regex==2021.10.23
-requests==2.26.0
 rfc3986==1.5.0
-six==1.16.0
-sniffio==1.2.0
+sniffio==1.3.0
 sortedcontainers==2.4.0
-starlette==0.16.0
-toml==0.10.2
-tomli==1.2.2
-trio==0.19.0
-typing-extensions==3.10.0.2
-urllib3==1.26.7
-uvicorn[standard]
-virtualenv==20.9.0
-watchgod==0.7
-websockets==10.0
-wrapt==1.13.2
+starlette==0.21.0
+tomli==2.0.1
+typing_extensions==4.4.0
```

Comment:
Is the a genuine reason why you decided to tamper with the requirements file?

Reply:
I was having some conflicts while trying to install dependencies

Reply:
Please, what happened here?
Some essential packages have been removed... pytest, bson, etc. Why?
Please kindly fix that?

Reply:
Okay. I'll fix it now